cluster/addons/dns/coredns/coredns.yaml.base

```diff
@@ -129,6 +129,8 @@ spec:
         - name: config-volume
           mountPath: /etc/coredns
           readOnly: true
+        - name: tmp
```

Comment:
is there a reason we need to explicitly define /tmp here?

Reply:
This is a cherry-pick from Kubernetes: https://github.com/kubernetes/kubernetes/pull/82128.
Arktos Coredns did have same issue before cherry-pick.